src/base/static/utils/collection-utils.js

```diff
@@ -34,15 +34,25 @@ const getModelFromUrl = ({ collections, route, mapConfig }) => {
   }
 };
 
-const createGeoJSONFromCollection = (collection, modelIdToFocus) => {
+const createGeoJSONFromCollection = ({
+  collection,
+  modelIdToFocus,
+  modelIdToHide,
+}) => {
   const features = collection.map(model => {
     const properties = Object.keys(model.attributes)
       .filter(key => key !== constants.GEOMETRY_PROPERTY_NAME)
       .reduce((geoJSONProperties, property) => {
         geoJSONProperties[property] = model.get(property);
         return geoJSONProperties;
       }, {});
-    properties.isFocused = modelIdToFocus === properties.id;
+
+    if (modelIdToHide === properties.id) {
+      return [];
```

Comment:
Filtering out the hidden models makes sense.

I'm not sure I understand the second suggestion-- the return type is just GeoJSON, with standard `geometry` and `properties` keys. Do you think it would be beneficial to document the two properties we add here (`isHidden` and `isFocused`)? Maybe something like:

```
properties: {
  isFocused?: bool,
  isHidden?: bool,
  // other properties...
}
```

Reply:
That's true, it's just GeoJSON. I think the `modelIdToFocus` and `modelIdToHide` args make this function a bit more confusing. But I think it's fine to punt for now, and we can address this when we fix our models.